packages/integrations/vue/src/index.ts

```diff
@@ -125,9 +126,11 @@ async function getViteConfiguration(
 
 	if (command === 'dev' && options?.devtools) {
 		const vueDevTools = (await import('vite-plugin-vue-devtools')).default;
+		const pluginOptions =  typeof options.devtools === 'object' ? options.devtools : {}
```

Comment:
```suggestion
		const devToolsOptions =  typeof options.devtools === 'object' ? options.devtools : {}
```
perhaps? I was confused what this was about reading the PR.